src/components/utils/Button.js

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import "./Button.css";
+import { Link } from "react-router-dom";
+
+export default function Button({ text, className = "", link }) {
+  return (
+    <button className={`btn ${className}`}>
+      {link && <Link to={link}>{text}</Link>}
+      {!link && { text }}
```

Comment:
삼항연산자 방식으로 써볼 수도 있겠네요 

```suggestion
      {link ? <Link to={link}>{text}</Link> : text}
```